pkg/webui/console/containers/schedule-delay-input/schedule-delay-input.js

```diff
@@ -0,0 +1,158 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import classnames from 'classnames'
+import bind from 'autobind-decorator'
+import { defineMessages } from 'react-intl'
+
+import Select from '../../../components/select'
+import Input from '../../../components/input'
+
+import PropTypes from '../../../lib/prop-types'
+
+import Notification from '../../../components/notification'
+import style from './schedule-delay-input.styl'
+
+const m = defineMessages({
+  miliseconds: 'miliseconds',
+  seconds: 'seconds',
+  minutes: 'minutes',
+  hours: 'hours',
+  durationWarning:
+    'Value of delay too low. The lower bound (130ms) will be used by the Gateway Server.',
+})
+
+const timeUnitOptions = [
+  { label: m.miliseconds, value: 'ms' },
+  { label: m.seconds, value: 's' },
+  { label: m.minutes, value: 'm' },
+  { label: m.hours, value: 'h' },
+]
+
+class ScheduleDelayInput extends React.PureComponent {
```

Comment:
This is not a container component, you're not connecting to the store or fetch anything.
Move this to `/components`